scripts/test-recommended-policy.sh

```diff
@@ -7,7 +7,9 @@ Usage:
   $(basename "$0")
 
 This script validates that file config/iam/recommended-policy-arn exists for a
-service controller repository.
+service controller repository and that each of the policies in the file are
+valid. If the file is empty, it will check for the existence of a 
```

Comment:
Instead of the file being empty, how about just checking for the existence of the recommended-policy-arn file and if not present, enforce the existence of an inline policy file?